packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -0,0 +1,56 @@
+import { IDropdownOption, IDropdownProps } from '../../Dropdown';
+import { IIconProps } from '../../Icon';
+
+export interface IComboBox {
+
+}
+
+export interface IComboBoxProps extends IDropdownProps {
+
+  /**
+   * Collection of options for this ComboBox
+   */
+  options?: IComboBoxOption[];
+
+  /**
+   * Callback issues when either:
+   * 1) the selected option changes
+   * 2) a manually edited value is submitted. In this case there may not be a matched option if allowFreeform is also true
+   *    (and hence only value would be true, the other parameter would be null in this case)
+   */
+  onChanged?: (option?: IComboBoxOption, index?: number, value?: string) => void;
+
+  /**
+   * Callback issued when the options should be resolved, if they have been updated or
+   * if they need to be passed in the first time
+   */
+  onResolveOptions?: () => IComboBoxOption[] | PromiseLike<IComboBoxOption[]>;
+
+  /**
+   * Whether the ComboBox is free form, meaning that the user input is not bound to provided items. Defaults to false.
+   */
+  allowFreeform?: boolean;
+
+  /**
+   * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of items. If
+   * the combo box is expanded, this will also scroll to the suggested item, and give it a selected style. Defaults to false.
+   */
+  autoComplete?: boolean;
+
+  /**
+   * Value to show in the input, does not have to map to a combobox option
+   */
+  value?: string;

```

Comment:
The scenario that requires this prop is the font size combobox, it allows freeform and will allow for input like "11.5" as valid, but will not add that value as an option in the menu.

Another case where is could be needed is when all the options are gotten at a later time than when the component is rendered. This allows the creator pass in the value to show without having 1) have any other info, 2) duplicate data just to make the rendering work